include/UIKit/UIImage.h

```diff
@@ -92,7 +96,9 @@ UIKIT_EXPORT_CLASS
 + (UIImage*)imageNamed:(NSString*)name
                          inBundle:(NSBundle*)bundle
     compatibleWithTraitCollection:(UITraitCollection*)traitCollection STUB_METHOD;
-+ (UIImage*)imageNamed:(NSString*)name; // Note, this caches the images somewhat like iPhone OS 2ish in that it never releases them. :)
++ (UIImage*)imageNamed:(NSString*)name; // Note, this caches the images
+// somewhat like iPhone OS 2ish in
+// that it never releases them. :)
```

Comment:
Will occur in a separate PR.

---
In reply to: [84808434](https://github.com/Microsoft/WinObjC/pull/1185#discussion_r84808434) [](ancestors = 84808434,84753537,84747216)